proteus/mprans/RANS3PF2D.h

```diff
@@ -1036,23 +1058,28 @@ namespace proteus
         duc_du = u/(uc+1.0e-12);
         duc_dv = v/(uc+1.0e-12);
         duc_dw = w/(uc+1.0e-12);
-        double fluid_velocity[2]={u,v}, solid_velocity[2]={u_s,v_s};
+        double fluid_velocity[2]={uStar,vStar}, solid_velocity[2]={uStar_s,vStar_s};
         double new_beta = closure.betaCoeff(1.0-phi_s,
 					    rho,
 					    fluid_velocity,
 					    solid_velocity,
-					    viscosity);
+					    viscosity)*DRAG_FAC;
+        //new_beta = 254800.0;//hack fall velocity of 0.1 with no pressure gradient
+        double beta2 = 156976.4;//hack, fall velocity of 0.1 with hydrostatic water
+        
+        mom_u_source += (1.0 - phi_s) * new_beta * (u - u_s) - TURB_FORCE_FAC*new_beta*nu_t*gradC_x/closure.sigmaC_  +
+          (1.0 - phi_s)*(1.0-DRAG_FAC)*beta2*(u-u_s);
+	mom_v_source += (1.0 - phi_s) * new_beta * (v - v_s) - TURB_FORCE_FAC*new_beta*nu_t*gradC_y/closure.sigmaC_ +
+          (1.0 - phi_s)*(1.0-DRAG_FAC)*beta2*(v-v_s);
```

Comment:
This is a hack, should be removed?

Reply:
I was going to leave these in for now. They are  useful in debugging changes to the formulation and numerics.